### ISSUE TITLE
Incompatibility with Combo Offers Combo Offers WooCommerce 2.10-3.0 plugin - Error message in the Checkout

### ISSUE THREAD

**Issue by Estalhun** [conedevelopment/simplepay-gateway#133]
- Simplepay Gateway bővítmény verzió: 2.6.0
- WooCommerce verzió: 8.0.3
- WordPress verzió: 6.3.1
- PHP verzió: 8.1.18

**Hibaüzenet a checkout-ban:** Hiba a fizetés feldolgozásakor. Kérlek próbáld újra.

Screenshot-ok  link: [https://drive.google.com/drive/folders/1MDimxbOr7oQRsptwMTJpVzP4VVZyzLzl?usp=sharing](https://drive.google.com/drive/folders/1MDimxbOr7oQRsptwMTJpVzP4VVZyzLzl?usp=sharing)

Termék link: [https://gallmet.hu/shop/gallmet-mix-90-db-sziver-90-db-egyutt-kedvezobben/](https://gallmet.hu/shop/gallmet-mix-90-db-sziver-90-db-egyutt-kedvezobben/)

Combo Offers WooCommerce: [https://wordpress.org/plugins/woo-combo-offers/](https://wordpress.org/plugins/woo-combo-offers/)

Simplepay-gateway log:
2023-09-14T08:24:02+00:00 INFO A kérelem érvénytelen: {"errorCodes":[5321],"salt":"2JuQs8ye3fLNcYfvUXs4jBi1xiTxleL3"}
2023-09-14T08:28:45+00:00 INFO A kérelem érvénytelen: {"errorCodes":[5321],"salt":"43CjOoLYsS6TJw6Voj2tKFhRwKbA9wUA"}

Fatal errors:
2023-09-14T08:53:00+00:00 CRITICAL Uncaught WpOrg\Requests\Exception\InvalidArgument: WpOrg\Requests\Cookie::__construct(): Argument #2 ($value) must be of type string, array given in /home/****/gallmet.hu/wp-includes/Requests/src/Exception/InvalidArgument.php:29
Stack trace:
#0 /home/****/gallmet.hu/wp-includes/Requests/src/Cookie.php(88): WpOrg\Requests\Exception\InvalidArgument::create()
#1 /home/****/gallmet.hu/wp-includes/class-wp-http.php(470): WpOrg\Requests\Cookie->__construct()
#2 /home/****/gallmet.hu/wp-includes/class-wp-http.php(352): WP_Http::normalize_cookies()
#3 /home/****/gallmet.hu/wp-includes/class-wp-http.php(634): WP_Http->request()
#4 /home/****/gallmet.hu/wp-includes/http.php(168): WP_Http->get()
#5 /home/****/gallmet.hu/wp-content/plugins/wp-seopress/inc/admin/ajax.php(131): wp_remote_get()
#6 /home/****/gallmet.hu/wp-includes/class-wp-hook.php(310): seopress_do_real_preview()
#7 /home/****/gallmet.hu/wp-includes/class-wp-hook.php(334): WP_Hook->apply_filters()
#8 /home/****/gallmet.hu/wp-includes/plugin.php(517): WP_Hook->do_action()
#9 /home/****/gallmet.hu/wp-admin/admin-ajax.php(188): do_action()
#10 {main}
  thrown hiba /home/****/gallmet.hu/wp-includes/Requests/src/Exception/InvalidArgument.php fájlban a 29 sorban

**Comment by iamgergo** [comment]
Hello,

Ennek nem tervezünk utánajárni. Ha van megoldásod, jöhet egy PR, azt megnézzük.

Köszi!

**Comment by Estalhun** [comment]
Hello,

Az "ennek" szó mit jelent? A Combo Offers kompatibilitás (több ilyen bővítmény létezik, tehát nem csak ezzel az egyel való kompatibilitást jeleni) vagy a Fatal Erros log-ban látható dolgokat? A Fatal Errors logot azért tettem be, hogy látszódjék, hogy nincs ilyen jellegű probléma a simplepay-gateway bővítménnyel.

Üdv:
István

**Comment by iamgergo** [comment]
@Estalhun A kompatibilitási problémára gondoltam. De úgy néz ki, hogy összefüggésben van a #134 -es issueval, így lehet, hogy ezt egy reverttel meg fogjuk oldani.
